build.gradle

```diff
@@ -92,7 +89,7 @@ dependencies {
     // Dependencies that library users should include in their job
     // shadow jar
     // --------------------------------------------------------------
-    implementation "org.alvearie.hri:hri-api-batch-notification:develop-SNAPSHOT"
+    implementation "org.alvearie.hri:hri-api-batch-notification:WHFHRI-693-SNAPSHOT"
```

Comment:
Don't forget to switch this back before merging.

Reply:
Switched ✅
For some reason when I was working on this before, it wasn't cooperating with the `develop-SNAPSHOT`.  But I switched it back and it's passing now.